app/initializers/responsive.js

```diff
@@ -1,2 +1,32 @@
-import responsive from 'ember-responsive/initializers/responsive';
-export default responsive;
+import Ember from 'ember';
+import config from '../config/environment';
```

Comment:
I think it is fine to not expose this initializer from the addon, I don't see a need to extend it, so it's likely someone would just do their own initialization.